src/main/python/rlbot/setup_manager.py

```diff
@@ -111,12 +117,21 @@ def run(self):
         self.game_interface.start_match()
         self.logger.info("Match has started")
 
-        self.logger.info("Press any character to exit")
+        instructions = "Press 'r' to reload all agents, or 'q' to exit"
+        self.logger.info(instructions)
         while not self.quit_event.is_set():
+            # Handle commands
             if msvcrt.kbhit():
-                msvcrt.getch()
-                self.shut_down()
-                break
+                command = msvcrt.getwch()
+                if command.lower() == 'r':  # reload
+                    self.reload_all_agents()
+                elif command.lower() == 'q':  # quit
```

Comment:
Could you listen for ESC also? Doesn't need to be in the instructions, I think it'll be convenient to have though.